recipes/bob.io.matlab/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "2.0.4" %}
+
+package:
+  name: bob.io.matlab
+  version: {{ version }}
+
+source:
+  fn: bob.io.matlab-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.io.matlab/bob.io.matlab-{{ version }}.zip
+  md5: 089155b2d864a6dca0f30491da0ed4fc
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  script: python -B setup.py install --single-version-externally-managed --record record.txt
```

Comment:
I tried to package `bob.io.matlab` without the `-B` option and here is what I get:
```
.
└── python3.5
    └── site-packages
        ├── bob
        │   ├── blitz
        │   │   └── __pycache__
        │   │       ├── extension.cpython-35.pyc
        │   │       └── __init__.cpython-35.pyc
        │   ├── core
        │   │   ├── __pycache__
        │   │   │   ├── __init__.cpython-35.pyc
        │   │   │   └── log.cpython-35.pyc
        │   │   └── random
        │   │       └── __pycache__
        │   │           └── __init__.cpython-35.pyc
        │   ├── extension
        │   │   └── __pycache__
        │   │       ├── boost.cpython-35.pyc
        │   │       ├── cmake.cpython-35.pyc
        │   │       ├── __init__.cpython-35.pyc
        │   │       ├── pkgconfig.cpython-35.pyc
        │   │       └── utils.cpython-35.pyc
        │   ├── io
        │   │   ├── base
        │   │   │   └── __pycache__
        │   │   │       └── __init__.cpython-35.pyc
        │   │   ├── matlab
        │   │   │   ├── bobskin.cpp
        │   │   │   ├── bobskin.h
        │   │   │   ├── data
        │   │   │   │   ├── test_1d_cplx.mat
        │   │   │   │   ├── test_1d.mat
        │   │   │   │   ├── test_2d_cplx.mat
        │   │   │   │   ├── test_2d.mat
        │   │   │   │   ├── test_3d_cplx.mat
        │   │   │   │   ├── test_3d.mat
        │   │   │   │   ├── test_4d_cplx.mat
        │   │   │   │   ├── test_4d.mat
        │   │   │   │   ├── test_cell.mat
        │   │   │   │   └── test.mat
        │   │   │   ├── file.cpp
        │   │   │   ├── file.h
        │   │   │   ├── __init__.py
        │   │   │   ├── _library.cpython-35m-x86_64-linux-gnu.so
        │   │   │   ├── main.cpp
        │   │   │   ├── __pycache__
        │   │   │   │   ├── __init__.cpython-35.pyc
        │   │   │   │   └── test.cpython-35.pyc
        │   │   │   ├── test.py
        │   │   │   ├── utils.cpp
        │   │   │   ├── utils.h
        │   │   │   ├── version.cpp
        │   │   │   └── version.cpython-35m-x86_64-linux-gnu.so
        │   │   └── __pycache__
        │   │       └── __init__.cpython-35.pyc
        │   └── __pycache__
        │       └── __init__.cpython-35.pyc
        └── bob.io.matlab-2.0.4-py3.5.egg-info
            ├── dependency_links.txt
            ├── not-zip-safe
            ├── PKG-INFO
            ├── requires.txt
            ├── SOURCES.txt
            └── top_level.txt
```
You can see that these `__pycache__` folders conflict with other `bob` packages.